tests/unit/test_plugins/test_sqlalchemy.py

```diff
@@ -1,13 +1,13 @@
-import advanced_alchemy
 from advanced_alchemy import base as sa_base
+from advanced_alchemy import filters as sa_filters
 from advanced_alchemy import types as sa_types
 from advanced_alchemy.extensions import litestar as sa_litestar
 
 from litestar.plugins import sqlalchemy
 
 
 def test_re_exports() -> None:
-    assert sqlalchemy.filters is advanced_alchemy.filters
+    assert sqlalchemy.filters is sa_filters
```

Comment:
I don't really understand why this is an issue all of a sudden - our pyright version is pinned, and `filters` wasn't named in `advanced_alchemy.__all__` in 0.7.4 either.